src/main/java/uk/gov/hmcts/darts/notification/service/impl/NotificationServiceImpl.java

```diff
@@ -61,12 +61,15 @@ public void scheduleNotification(SaveNotificationToDbRequest request) {
         List<String> emailAddresses = getEmailAddresses(request);
         String templateParamsString = getTemplateParamsString(request);
         for (String emailAddress : emailAddresses) {
-            saveNotificationToDb(
+            NotificationEntity notificationEntity = saveNotificationToDb(
                 request.getEventId(),
                 request.getCaseId(),
                 StringUtils.trim(emailAddress),
                 templateParamsString
             );
+            log.info("Notification scheduled: notificationId={}, type={}, caseId={}, status={}",
```

Comment:
Dave wants these notifications to be run in referenced in uk.gov.hmcts.darts.log.api.LogApi.